composer.json

```diff
@@ -43,14 +42,15 @@
         "webmozart/assert": "^1.11"
     },
     "require-dev": {
-        "nette/robot-loader": "^4.0",
         "phpstan/extension-installer": "^1.3.1",
         "phpstan/phpstan-phpunit": "^1.3.15",
         "phpstan/phpstan-webmozart-assert": "^1.2.2",
         "phpunit/phpunit": "^10.5",
         "rector/rector-generator": "^0.7.10",
         "rector/release-notes-generator": "dev-main",
+        "shipmonk/composer-dependency-analyser": "^1.4",
         "spatie/enum": "^3.13",
+        "symfony/yaml": "^6.4",
```

Comment:
why needed?

Reply:
It is used in `rules-tests/Removing/Rector/ClassMethod/ArgumentRemoverRector/config/configured_rule.php`. 

I think we can remove it if you want. Using `::class` would work even if such transitive dependency would disappear one day.

Reply:
The test should work with any class. It would be best to remove the yaml one and create a dummy on in `Source`